tests/instruments/yokogawa/test_aq6370d.py

```diff
@@ -0,0 +1,404 @@
+import pytest
+
+from pymeasure.instruments.yokogawa.aq6370 import AQ6370D
```

Comment:
I think you have to add the `d` again:
```suggestion
from pymeasure.instruments.yokogawa.aq6370d import AQ6370D
```